docs/api/plugins.md

```diff
@@ -439,6 +446,23 @@ uniform request distribution.  To enable this, you can pass in
 `options.tokensTable`, which is simply any Object that supports `put` and `get`
 with a `String` key, and an `Object` value.
 
+## Inflight Request Throttling
+
+```js
+const opts = { limit: 600, server: server };
+opts.error = new Error('Resource Exhausted');
+error.statuscode = 420;
```

Comment:
Should that be camelCase `statusCode`? Also, I might suggest adding an example using restify-errors constructors.